packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -88,6 +92,7 @@ export function useHiddenSelect<T>(props: AriaHiddenSelectProps, state: SelectSt
       size: state.collection.size,
       value: state.selectedKey ?? '',
       onChange: (e: React.ChangeEvent<HTMLSelectElement>) => state.setSelectedKey(e.target.value)
```

Comment:
Looks like there's a missing comma:

```suggestion
      onChange: (e: React.ChangeEvent<HTMLSelectElement>) => state.setSelectedKey(e.target.value),
```